packages/snap/snap.manifest.json

```diff
@@ -7,7 +7,7 @@
     "url": "git+https://github.com/MetaMask/snap-simple-keyring.git"
   },
   "source": {
-    "shasum": "s5XnCGE4PLuRg5wcW7QyK3a+FE8P3zfJLemiz6t86O4=",
+    "shasum": "KvlHpqgEHTR53aD3rwHq9e5+9ep25+pjEhEBFmAgFes=",
```

Comment:
should these changes be committed?

Reply:
Yes, if the source code changes the shasum should also change